Update README.md (#2)
linuxdeepin/deepin-gomoku: README.md

## deepin-gomoku
Gomoku is an easy and funny chess game that is suitable for all ages.
五子棋是一款容易上手，老少皆宜，而且趣味横生，引人入胜的棋类小游戏。

### Core
The GUI is based on DTK, Qt (Supported >= 5.12).

Build-Depends: debhelper (>= 11),cmake, pkg-config, qtbase5-dev, qtchooser (>= 55-gc9562a1-1~),qt5-qmake, libdtkwidget-dev, qttools5-dev-tools, libqt5svg5-dev, qttools5-dev,libgtest-dev, libgmock-dev

## Installation
sudo apt-get install deepin-gomoku

### Build
mkdir build
cd build
cmake ..
make
make install

## Getting help

- [Official Forum](https://bbs.deepin.org/) for generic discussion and help.
- [Developer Center](https://github.com/linuxdeepin/developer-center) for BUG report and suggestions.
- [Wiki](https://wiki.deepin.org/)

## Getting involved

We encourage you to report issues and contribute changes

- [Contribution guide for developers](https://github.com/linuxdeepin/developer-center/wiki/Contribution-Guidelines-for-Developers-en) (English)
- [开发者代码贡献指南](https://github.com/linuxdeepin/developer-center/wiki/Contribution-Guidelines-for-Developers) (中文)
- [Translate for your language on Transifex](#) *please update to the actual Transifex link of this project*

## License

> License description here. The license name is suggested to use the same one as [SPDX license identifier](https://spdx.org/licenses). Following is an example:
> 
> Deepin Example is licensed under [GPL-3.0-or-later](LICENSE)

Reviewer: ```suggestion
Deepin Gomoku is licensed under [GPL-3.0-or-later](LICENSE)
```

上面”Deepin 示例“肯定要改成符合项目描述的名称的呀 😂

## deepin-gomoku

Gomoku is an easy and funny chess game that is suitable for all ages.

五子棋是一款容易上手，老少皆宜，而且趣味横生，引人入胜的棋类小游戏。

### Core

The GUI is based on DTK, Qt (Supported >= 5.12).

 

Build-Depends: debhelper (>= 11),cmake, pkg-config, qtbase5-dev, qtchooser (>= 55-gc9562a1-1~),qt5-qmake, libdtkwidget-dev, qttools5-dev-tools, libqt5svg5-dev, qttools5-dev,libgtest-dev, libgmock-dev

## Installation

sudo apt-get install deepin-gomoku

### Build

mkdir build

cd build

cmake ..

make

make install

## Getting help

> \- [Official Forum](https://bbs.deepin.org/) for generic discussion and help.

Reviewer: 列表的前面不用加 `\`，下同。
```suggestion
- [Official Forum](https://bbs.deepin.org/) for generic discussion and help.
```

\- [Developer Center](https://github.com/linuxdeepin/developer-center) for BUG report and suggestions.

> 

Reviewer: 列表组之间可以不留空行

\- [Wiki](https://wiki.deepin.org/)

> 

Reviewer: 太多多余的空行了，段落之间表示分段的话只需要一个空行就可以了。其它地方同。
